src/api/json_tokenize.hpp

```diff
@@ -0,0 +1,18 @@
+#ifndef MIGRAPHX_GUARD_API_RTGLIB_JSONIZE_ATTR_STRING_HPP
+#define MIGRAPHX_GUARD_API_RTGLIB_JSONIZE_ATTR_STRING_HPP
+
+#include <string>
+#include <migraphx/config.hpp>
+
+namespace migraphx {
+inline namespace MIGRAPHX_INLINE_NS {
+
+using siter = std::string::const_iterator;
+using token = std::pair<siter, siter>;
```

Comment:
These should just be moved to the .cpp file.

Reply:
Done